examples_tests/51.RadixSort/Histogram.comp

```diff
@@ -43,7 +42,7 @@ void main()
 		nbl_glsl_workgroupBallot(predicate);
 		local_histogram[i] = nbl_glsl_workgroupBallotInclusiveBitCount();
 
-		// Todo: What is the problem with dynamically indexing local variables?
+		// Question: What is the problem with dynamically indexing local variables?
```

Comment:
they will get promoted to `shared` or spilled to VRAM anyway, because most GPUs cannot index registers